microsoft-teams/1.0/antora.yml

```diff
@@ -0,0 +1,17 @@
+name: microsoft-teams-connector
```

Comment:
edit instructions

Scope: index.adoc ONLY (the rest of the files are template and reference)
Type: CE
Due: EOD tomorrow (Th) if possible